### ISSUE TITLE
[Crash] Version 1.13.0 crash on initialization !

### ISSUE THREAD

**Issue by parlinsqe** [amplitude/Amplitude-Kotlin#175]
## Expected Behavior
Amplitude should be able to initiated correctly

## Current Behavior
It is crash when initiating Amplitude

`No direct method <init>(Ljava/lang/String;Landroid/content/Context;IILjava/lang/String;ZLcom/amplitude/core/StorageProvider;Lcom/amplitude/core/LoggerProvider;Ljava/lang/Integer;Ljava/lang/String;Lkotlin/jvm/functions/Function3;IZLcom/amplitude/core/ServerZone;Ljava/lang/String;Lcom/amplitude/core/events/Plan;Lcom/amplitude/core/events/IngestionMetadata;ZZZLcom/amplitude/android/TrackingOptions;ZZZJZLcom/amplitude/android/DefaultTrackingOptions;JLcom/amplitude/core/StorageProvider;Lcom/amplitude/id/IdentityStorageProvider;ZILkotlin/jvm/internal/DefaultConstructorMarker;)V in class Lcom/amplitude/android/Configuration; or its super classes (declaration of 'com.amplitude.android.Configuration' appears in /data/app/~~SPSRcroseqPOM31PApKBFg==/com.smmf.dealer-t0p3yj8HP1Mqh7GApJ6a_w==/base.apk)`

## Possible Solution
Is it possible to revert back to v1.12.2 until 1.13.0 stable ?

## Steps to Reproduce
1. Use version v1.13.0
2. Initiate amplitude
```
amplitude = Amplitude(
                    Configuration(
                        apiKey = apiKey,
                        context = applicationContext,
                        instanceName = INSTANCE_NAME
                    )
                )
```

## Environment
- Samsung A32
- Emulator with

**Comment by izaaz** [comment]
Thank you for raising this issue @parlinsqe We are looking into this.

**Comment by Mercy811** [comment]
Hi @parlinsqe, could you give us more context. I can run it successfully locally. 

Kotlin app
<img width="1603" alt="Screenshot 2024-01-24 at 10 06 13" src="https://github.com/amplitude/Amplitude-Kotlin/assets/31029607/12752f75-21bd-49b6-a1a8-b1adf109e96a">

Java app

<img width="1607" alt="Screenshot 2024-01-24 at 10 12 36" src="https://github.com/amplitude/Amplitude-Kotlin/assets/31029607/2d63cf92-9734-4202-aec3-75bf2687c2cc">

**Comment by Mercy811** [comment]
Find a similar [error](https://youtrack.jetbrains.com/issue/KT-32417/Kotlin-Android-NoSuchMethodError-after-add-more-parameters-with-default-argument). This error was solved by adding `@JvmOverloads` annotation to the modified function.

`v1.13.0` adds `offline` to 'com.amplitude.android.Configuration' and `com.amplitude.core.Configuration`. Both of their constructor have `@JvmOverloads` 
- https://github.com/amplitude/Amplitude-Kotlin/blob/1ee97e60cd7859755232c3bc7f05693ba6716561/android/src/main/java/com/amplitude/android/Configuration.kt#L16
- https://github.com/amplitude/Amplitude-Kotlin/blob/1ee97e60cd7859755232c3bc7f05693ba6716561/core/src/main/java/com/amplitude/core/Configuration.kt#L13

Could you try to rebuild your app?

**Comment by Mercy811** [comment]
Hi @parlinsqe, are you still blocked?

**Comment by parlinsqe** [comment]
Hi @Mercy811 yes, we are still blocked, but we have decided to use `1.12.2` for now

**Comment by Mercy811** [comment]
Hi @parlinsqe, have you tried clean and rebuild your project?

**Comment by Mercy811** [comment]
Did you sync gradle?

**Comment by parlinsqe** [comment]
Yes, I did, as I can see in the external libraries section in android studio, it shown amplitude with version 1.13.0. And it crashing

**Comment by Mercy811** [comment]
Hi @parlinsqe, could you provide a minimum project reproduce this issue.

**Comment by izaaz** [comment]
@parlinsqe thanks for raising this issue. We've been monitoring this library on our end and haven't heard any other complaints. We also see data coming in from this version from other apps. 

I am closing this issue for now but if you have any other information on why this may not be working for you, please feel free to reopen this issue and we'll jump right in.
